juntagrico_billing/mailer.py

```diff
@@ -13,8 +13,7 @@ def send_bill_share(bill, share, member):
         'member': member,
         'bill': bill,
         'share': share,
-        'protocol': 'https',
-        'serverurl': Site.objects.get_current().domain
+        'serverurl': 'https' + Site.objects.get_current().domain
```

Comment:
Isn't the `://` missing now? 
Should be `'https://' + Site.objects.get_current().domain` to yield the same result as before.